src/mainwindow.cpp

```diff
@@ -1348,8 +1169,8 @@ void MainWindow::showEvent(QShowEvent *event)
   QMainWindow::showEvent(event);
 
   if (!m_WasVisible) {
+    ui->modList->refreshFilters();
```

Comment:
`readSettings()` is now called after `refreshFilters()`, was that intended? Might not matter.

Reply:
It's intended because before `refreshFilters`, the filter list is empty so the reload from the settings does not work.